sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -254,3 +255,46 @@ case class Pow(left: Expression, right: Expression)
       """
   }
 }
+
+case class Pmod(left: Expression, right: Expression) extends BinaryArithmetic {
```

Comment:
This expression looks different than other math expressions, and it extends `BinaryArithmetic`, not `BinaryMathExpression`, I think we should move it to `arithmetic.scala`.